chapter_builders-guide/parameters.md

```diff
@@ -143,6 +181,13 @@ That's why we need to request the value explicitly.
 In addition to the value, each parameter also allows us to access the gradient. Because we have not invoked backpropagation for this network yet, it is in its initial state.
 :end_tab:
 
+:begin_tab:`jax`
+Unlike the other frameworks, JAX doesn't keep a track of the gradients over the
+neural network parameters, instead the params and the network are decoupled.
```

Comment:
Spell properly in main text unless they are variable names surrounded with accents.
params -> parameters